example_rails7/Gemfile

```diff
@@ -68,4 +68,4 @@ group :test do
   gem "webdrivers"
 end
 
-gem 'rolemodel_rails', '~> 0.7.0', group: :development, path: '..'
+gem "rolemodel_rails", "~> 0.8.0", :group => :development, :path => ".."
```

Comment:
Is there a reason for the change to double quotes instead of single?

Reply:
Nope. It was autogenerated

Reply:
```suggestion
gem 'rolemodel_rails', '~> 0.8.0', :group => :development, :path => '..'
```